jobbergate-agent/pyproject.toml

```diff
@@ -20,13 +20,12 @@ packages = [{ include = "jobbergate_agent" }]
 "Changelog" = "https://github.com/omnivector-solutions/jobbergate/blob/main/jobbergate-agent/CHANGELOG.rst"
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.8.1"
 jobbergate-core = { path = "../jobbergate-core", develop = true }
 apscheduler = "3.10.3"
 httpx = "0.24.1"
 importlib-metadata = { version = "^6.8.0", python = "<3.10" }
 loguru = "0.6.0"
-pendulum = "^2.1.2"
```

Comment:
Is it missing `pendulum = "3.0.0b1"` ?

Reply:
It is a good points, but it is already on jobbergate-core, so I cut off the duplication